### PR TITLE
Clean up and document s5_children recipe

### PR DESCRIPTION
**Pull request by rkjaran**
These changes include fixes to the s5_children recipe that adapt the scripts to
the published versions of the corpora. They also document how to run the recipe
using the Docker image in this repository.

### CODE REVIEW

File: s5_children/run.sh [hunk 1]
```diff
@@ -13,6 +13,14 @@ nj_train=20
 nj_decode=32
 stage=0
 
+# NOTE! In the future the ASR data, LM training text and pronunciation dictionary
+# will be downloaded from online first, e.g. Clarin
+samromur_root=/data/asr/samromur/samromur_ldc
```
Comment (lumpidu) [inline s5_children/run.sh:18]: are these absolute paths mapped to the Docker container when starting the training ? I couldn't see those mapped there inside the `Readme.md` ?

File: s5_children/README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+           -it samromur-asr/kaldi:gpu-latest \
+           --gpus all \
+           ./run.sh --samromur-root /datasets/samromur_21.05 \
+                    --samromur-teen /datasets/samromur_children_21.09 \
+                    --lm-train lm_training_text.txt --prondict-orig prondict.txt
+```
+
+If the recipe finishes successfully you'll end up with a Kaldi chain model in
+`exp/chain/tdnn_7n_sp`.
+
```
Comment (lumpidu) [inline s5_children/README.md:42]: would it be too complex to also provide some clues about inferencing and evaluation ?

File: s5_children/README.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+# TDNN recipe for Samrómur 21.05 and Samrómur Children 21.09
+
+The file [./run.sh](./run.sh) contains a recipe for training a Kaldi chain TDNN
+model from the published Samrómur and Samrómur Children datasets. The recipe
+assumes you have already downloaded both datasets, have available training texts
+for a language model and a base pronunciation dictionary that uses the same
+phoneme set as the Sequitur G2P model in
```
Comment (lumpidu) [inline s5_children/README.md:7]: - Aren't these training texts already available somewhere ?
- Is the language model trained within these training steps ?
- Is this language model a KenLM model ?

**Comment by lumpidu** [comment]
I'd suggest to also update the top-level `Readme.md` for expressing the now better state of the `s5_children` recipe.

File: s5_children/run.sh (hunk 1 above)
Comment (rkjaran) [reply s5_children/run.sh:18]: Not these absolute paths but these values of `samromur_root` and `samromur_teen` are overridden with arguments in lines 25 and 26 in README.md and both are assumed to exist in the mapped in /datasets directory.

File: s5_children/README.md (hunk 2 above)
Comment (rkjaran) [reply s5_children/README.md:42]: For inference on a single audio file see: https://github.com/icelandic-lt/samromur-asr/pull/5/commits/24dbb3baabafc555f3a4f7df8270c0be39d9e916#diff-6455f0aa057de10b8256d040340e4ffe00b66842cb54fd8af0f0725aadc0f81fR41

File: s5_children/README.md (hunk 3 above)
Comment (rkjaran) [reply s5_children/README.md:7]: 1. I've not been able to get the exact training texts referenced in the recipe. They should be derived from the IGC
2. Yes. I've added a mention of it in README.md
3. Yes

**Pull request merged**